Automattic/wp-calypso
# Podcasting: Update settings layout and style

This PR makes some layout and style adjustments to the Podcast Details screen to help focus the user on required actions and podcasting state. It moves the category selector to the top of the page, the disabled button to the bottom, updates the disabled scrim, and floats the title and subtitle beside the cover image.

This PR is part of a larger epic and is behind the manage/site-settings/podcasting feature flag. See p3Ex-32D-p2.

**Before** | **After**
------------ | -------------
<img width="1207" alt="before-disabled" src="https://user-images.githubusercontent.com/942359/41368171-1e931ce2-6f0f-11e8-82e4-fc1a52f995e5.png"> | <img width="1208" alt="after-disabled" src="https://user-images.githubusercontent.com/942359/41368181-2b530b54-6f0f-11e8-82c6-838cf4b63c15.png">
<img width="1207" alt="before-enabled" src="https://user-images.githubusercontent.com/942359/41368210-4689c2d2-6f0f-11e8-94a5-12149855d4f0.png"> | <img width="1209" alt="after-enabled" src="https://user-images.githubusercontent.com/942359/41368194-36c71b38-6f0f-11e8-96f9-a93d9751a3c1.png">

**Update:**
This now addresses some mobile styling changes that float the cover image setting buttons on smaller screens and limit the drop-down widths.

<img width="369" alt="mobile-update" src="https://user-images.githubusercontent.com/942359/41419565-597c9ebc-6fc0-11e8-80a6-616352fce454.png">

**To test:**
- With the `manage/site-settings/podcasting` enabled (default in development):
- Visit the Settings > Writing > Enable/Manage Podcast
- Verify that things look and behave as expected/shown in screenshots.
